src/applications/personalization/view-dependents/manage-dependents/tests/e2e/manage-dependents.cypress.spec.js

```diff
@@ -0,0 +1,91 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import mockDependents from '../fixtures/mock-dependents.json';
+import mockUser from '../fixtures/mockUser.json';
+import { rootUrl } from '~/applications/personalization/view-dependents/manifest.json';
+
+const VIEW_DEPENDENTS_PATH = rootUrl;
+const VIEW_DEPENDENTS_ENDPOINT = '/v0/dependents_applications/show';
+const FORM_686_ENDPOINT = '/v0/dependents_applications';
+
+const setup = () => {
+  disableFTUXModals();
+  cy.login(mockUser);
+  cy.intercept('GET', VIEW_DEPENDENTS_ENDPOINT, mockDependents).as(
+    'mockDependents',
+  );
+  cy.intercept('GET', '/v0/feature_toggles*', {
+    data: {
+      type: 'feature_toggles',
+      features: [
+        {
+          name: 'dependents_management',
+          value: true,
+        },
+      ],
+    },
+  });
+  cy.visit(VIEW_DEPENDENTS_PATH);
+};
+
+const testAxe = () => {
+  cy.injectAxe();
+  cy.axeCheck();
+};
+
+const testInlineValidation = async () => {
+  const buttons = await cy.findAllByRole('button', {
+    name: /Remove this dependent/,
+  });
+  expect(buttons).to.have.length(2);
+  buttons[0].click();
+  await cy
+    .findByRole('button', {
+      name: /Submit VA Form 686c to remove this dependent/,
+    })
+    .click();
+  await cy.findByText(/Please select an option/);
```

Comment:
I was wondering if this test is an adequate test of if you need to do `cy.findByText(/Please select an option/).should('exist')` so I tried to test is locally and discovered that the tests aren't passing for me :( 
<img width="1480" alt="Screen Shot 2021-08-13 at 3 28 43 PM" src="https://user-images.githubusercontent.com/20728956/129424135-bb0a7257-92e8-49f9-85e4-0ce779bd8dd0.png">

Reply:
Turns out the mock data I was using wasn't the right shape...but the tests were passing for me locally because I had mocked some redux stuff directly and the tests were using that instead 😬